src/components/client/notifications/RepoNotificationSetting.tsx

```
@@ -0,0 +1,143 @@
+        method: "PUT",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({
+          ignoredTypes: payload.types,
+        }),
+      });
+      if (!res.ok) throw new Error("更新失敗");
+      return res.json();
+    },
+    onMutate: async (payload: { types: string[] }) => {
+      await queryClient.cancelQueries({ queryKey: ["repositories"] });
+      const previousData = queryClient.getQueryData<any[]>(["repositories"]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)